study.md

```diff
@@ -43,5 +57,19 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+  A hash URL makes sure that starting state of the app is loaded and kept up to
+date as you move around and interact with the app.
+
+  When deploying to GH pages, it's important to make sure your environment
+variables point to the proper project name and the proper location API for
+URLS. Also make sure to load all necessary file paths into the /dist folder in
+order for gh pages to properly build your app.
+
+  To use the 'history API instead of hash change your config/environment.js file
+and set the location to history like so:
+
+if (environment === 'production') {
+  ENV.baseURL = '/project-name';
+  ENV.locationType = 'history';
+}
```

Comment:
You also cannot use the history API if you're hosting on GH Pages. This is because GH pages isn't running an Ember server, it's running a static page server, and can only serve up pages that have been hard-coded into the project. Ember sidesteps this by loading off of the root page and not loading any other pages; instead, it breaks apart the URL and uses it to change the view state without loading a new page.